rules/better-regex.js

```diff
@@ -111,6 +111,9 @@ const schema = [
 	},
 ];
 
+/**
+ * @type {import('eslint').Rule.RuleModule}
+ */
```

Comment:
```suggestion
/**
@type {import('eslint').Rule.RuleModule}
*/
```

I prefer this style. I find the `*` inside the comment to be pure noise and makes editing harder than it should be.

Reply:
Updated, that's fine with me. It can also be a single line comment if desired.

Reply:
Updated again to single-line. No need for it to take up additional space.